KitchenSink/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,6 +1,16 @@
 {
   "images" : [
     {
+      "idiom" : "iphone",
```

Comment:
I'm not sure. I'm sure we didn't do it manually. Part of the project upgrade process perhaps?